src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -142,6 +143,12 @@ class WorkspaceReimburseView extends React.Component {
         }
 
         const distanceCustomUnit = _.find(lodashGet(this.props, 'policy.customUnits', {}), unit => unit.name === 'Distance');
+        if (!distanceCustomUnit) {
+            Log.warn('Policy has no customUnits, returning early.', {
+                policyID: this.props.policyID,
+            });
+            return;
+        }
```

Comment:
Can we be certain that this is where the error occurs? I see a couple more calls to `updateWorkspaceCustomUnit`.

Reply:
This is the only place I see us calling `Policy.updateWorkspaceCustomUnit`, am I crazy?

Reply:
Looking at the [Firebase stack trace,](https://console.firebase.google.com/u/0/project/expensify-chat/crashlytics/app/android:com.expensify.chat/issues/9169b0f2a0ed276fad34337d803104de?time=last-seven-days&versions=1.1.99-2%20(1001019902)&sessionEventKey=631B202A034A00014D11FDBA7DC394EC_1719782693429934498) it seems that the third param of this function is the undefined object...

Reply:
You mean `newCustomUnit`?

In the Firebase stack trace I see "Cannot read property 'customUnitID' of undefined", and we do have `currentCustomUnit.customUnitID` and `newCustomUnit.customUnitID` - though I don't see how `newCustomUnit` can be undefined since it's always passed in here, right?

https://github.com/Expensify/App/blob/2343eeb08fbb89ec34e3de331ae1f75e494478fb/src/pages/workspace/reimburse/WorkspaceReimburseView.js#L147-L149

Reply:
> You mean newCustomUnit?

Oh yeah. It's odd to me too, but I agree your check should work. I worry a bit that for some reason an empty object is passed to as the param, but again -- no idea how that could be occuring.